x-pack/plugins/upgrade_assistant/server/lib/reindexing/reindex_service.test.ts

```diff
@@ -611,11 +612,11 @@ describe('reindexService', () => {
           });
         });
 
-        it('fails if docs created is less than count in source index', async () => {
```

Comment:
Deleted this test, leftover from when this behaviour was removed:

https://github.com/elastic/kibana/pull/124571#discussion_r798862107